build/config.gypi

```diff
@@ -0,0 +1,72 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
Do we need this file? 🤔

Reply:
I have delete that file. I didn´t know what is it.